src/test/scala/io/iohk/ethereum/vm/CreateOpcodeSpec.scala

```diff
@@ -233,7 +243,26 @@ class CreateOpcodeSpec extends WordSpec with Matchers {
     "refund the correct amount of gas" in {
       result.stateOut.gasRefund shouldBe result.stateOut.config.feeSchedule.R_selfdestruct
     }
+  }
+
+  "initialization includes REVERT opcode" should {
```

Comment:
there should also be the test case that `error message is available in the returndata buffer.` (one of my earlier comments)

Reply:
But we don't have at the moment returndata buffer functionality. It will be in the next pr.

Reply:
oo my bad, i did not raealize that this new opcodes https://github.com/ethereum/EIPs/blob/master/EIPS/eip-211.md are called return buffer